proto/money/rpc/reservation_folio_svc.proto

```diff
@@ -179,6 +194,10 @@ service ReservationFolioSvc {
 		returns(ReservationFolioSvcGetOnFileCardsResponse);
 	rpc VoidSale(ReservationFolioSvcVoidSaleRequest)
 		returns(ReservationFolioSvcVoidSaleResponse);
+	rpc IncreaseAuthorizationAmount(ReservationFolioSvcAuthorizationModificationRequest)
```

Comment:
The naming could use a little work here

`IncreaseAuthorizationAmount` is ok. It sounds like a database CRUD op, when in reality we're going all the way to the processor. You might want to call this `RequestIncrementalIncrease` or `RequestAuthorizationIncrease` to connote the possibility of failure.

I don't like `ReverseAuthorizationAmount`, if you want to keep it simple and consistent with industry terminology you might say `ReverseAuthorization`, if you think that's too similar to void, then maybe `RequestAuthorizationReduction`.

Given the inherent dualism of "increase" vs "decrease", you could even do an `AdjustAuthorizedAmount`, further suggested by the two methods' exact duplication of parameters as-written. To the extent possible, I'd like to insulate the client code from the [night watch](https://www.usenix.org/system/files/1311_05-08_mickens.pdf) that is card processing. 

Write back and let me know what you think, I think I'm leaning toward treating it as a single endpoint at this pt.